rust/Cargo.toml

```diff
@@ -74,8 +74,8 @@ bindgen=[]
 # Patched in
 [patch.crates-io]
 cid = { git = "https://github.com/multiformats/rust-cid", branch = "steb/cbor-hack" }
-fvm = { git = "https://github.com/filecoin-project/ref-fvm", branch = "steb/extern" }
-fvm_shared = { git = "https://github.com/filecoin-project/ref-fvm", branch = "steb/extern" }
+fvm = { path = '../../../../fvm/fvm' }
```

Comment:
Drop after https://github.com/filecoin-project/ref-fvm/pull/330 lands, but not sure what to patch to? master? no patch at all?

Reply:
I think we can drop the patch entirely and just depend on the git version.